packages/core/src/asyncContext/stackStrategy.ts

```diff
@@ -132,9 +132,12 @@ export class AsyncContextStack {
  */
 function getAsyncContextStack(): AsyncContextStack {
   const registry = getMainCarrier();
+
+  // For now we continue to keep this as `hub` on the ACS,
+  // as e.g. the Loader Script relies on this.
+  // Eventually we may change this if/when we update the loader to not require this field anymore
   const sentry = getSentryCarrier(registry) as { hub?: AsyncContextStack };
```

Comment:
we should probably link to the other pice of the puzzle: https://github.com/getsentry/sentry-javascript/blob/f1c4611c551816ca2fad537a6b5a17ead1b75b16/packages/core/src/sdk.ts/#L50-L62

```suggestion
  // Eventually we may change this if/when we update the loader to not require this field anymore
  // Related, we also write to `hub` in {@link ./sdk.ts registerClientOnGlobalHub} 
  const sentry = getSentryCarrier(registry) as { hub?: AsyncContextStack };
```

Reply:
which reminds me that the comment in the linked code is also a bit outdated with the renaming in #11630